ibm_cloud_sdk_core/utils.py

```diff
@@ -16,14 +16,33 @@
 # from ibm_cloud_sdk_core.authenticators import Authenticator
 import datetime
 import json as json_import
+import ssl
 from os import getenv, environ, getcwd
 from os.path import isfile, join, expanduser
 from typing import List, Union
 from urllib.parse import urlparse, parse_qs
 
+from requests.adapters import HTTPAdapter
+from urllib3.util.ssl_ import create_urllib3_context
+
 import dateutil.parser as date_parser
 
 
+class SSLHTTPAdapter(HTTPAdapter):
+    """Wraps the original HTTP adapter and adds additional SSL context.
+    """
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    #pylint: disable=arguments-differ
+    def init_poolmanager(self, connections, maxsize, block):
+        """Extends the parent's method by adding minimum SSL version to the args.
+        """
+        ssl_context = create_urllib3_context()
```

Comment:
I was trying to understand the difference between this and `ssl.create_default_context()`. It appears the `urllib3` version is modified to remove some insecure older things so it seems like using this route to get a context is good anyway. Was there any other reason?

Reply:
No other reason. I found it a bit more secure, that's all.